build/Jamfile.v2

```diff
@@ -136,8 +137,18 @@ local cxx_requirements = [ requires
     cxx11_override
 ] ;
 
-project boost/filesystem
+constant boost_dependencies_private :
+    /boost/atomic//boost_atomic
+    /boost/core//boost_core
+    /boost/predef//boost_predef
+    /boost/scope//boost_scope
+    /boost/winapi//boost_winapi
+    ;
```

Comment:
Boost.Atomic and Boost.WinAPI are conditional dependencies. For Boost.Atomic, see `check-cxx20-atomic-ref` rule.